GoogleForCreators/web-stories-wp
# Quick Actions: Remove panel highlight when entering trim mode

## Context

This removes the highlighting of the video settings panel when clicking "Trim video" in the quick-actions menu.

## User-facing changes

_Highlight no longer happens, so nothing to show here._

## Testing Instructions

This PR can be tested by following these steps:

1. Add a video to the canvas
2. Click the "Trim video" option in the qwuick-actions menu
3. Observe that trim mode is activated without the video settings panel highlights (i.e. _flashes a white overlay briefly_)

## Checklist




---

Fixes #9183
